apps/src/experiments.js

```diff
@@ -16,29 +21,61 @@ experiments.getQueryString_ = function () {
   return window.location.search;
 };
 
+experiments.getEnabledExperiments = function () {
+  var jsonList = localStorage.getItem(STORAGE_KEY);
+  var enabled = jsonList ? JSON.parse(jsonList) : [];
+  OLD_KEY_WHITELIST.forEach(function (key) {
+    if (localStorage.getItem('experiments-' + key) === 'true') {
+      enabled.push(key);
+    }
+  });
+  return enabled;
+};
+
+experiments.setEnabled = function (key, shouldEnable) {
+  var allEnabled = this.getEnabledExperiments();
+  if (!allEnabled.includes(key) && shouldEnable) {
+    allEnabled.push(key);
+  } else if (allEnabled.includes(key) && !shouldEnable) {
```

Comment:
I beleve `includes` is ES6. Presumably babelify is providing a polyfill, but it might be worth a quick test that this works in a browser that doesn't support `includes` natively (IE).